### DISCUSSION TITLE
How to check that backup works on iOS?

### DISCUSSION THREAD

**Discussion by marioklaver** [react-native-async-storage/async-storage#512] [Q&A]
## You want to:
I want to make sure the data in my application is backupped.  

## Details:
I created a new react native application. Stored some stuff in asyncStorage and launched the application from XCode onto my phone. I thought the proper way to validate that backup is working is by going to General > iPhone Storage and 'Offload App'. But when I click my application, I don't see the option 'Offload App'. I only see 'Delete App'. I use react native 0.63.3 and @react-native-async-storage/async-storage version 1.13.2 I also set the key RCTAsyncStorageExcludeFromBackup to false.

Do you know what I did wrong? Or how can I otherwise check that a backup is stored in iCloud?

**Comment by tido64** [comment]
Have you enabled iCloud capabilities in your app? Please have a look at Apple's documentation for how to do so: https://help.apple.com/xcode/mac/current/#/dev52452c426

**Reply by marioklaver** [reply]
Hi Tommy, thanks for your reply. I just tried it. But with no luck. 

Hereby a super simple repository to show you that it is not working:
https://github.com/marioklaver/async-storage-backup

I ran this project from xcode on my iphone 8. Also tried on my ipad with no luck.

Can you see what I did wrong? If you found out, I am happy to help updating the documentation to help other people with the same problem.

Regards
Mario

**Reply by marioklaver** [reply]
Hi Tommy, did you have time to take a look at the example code? Maybe it's a bug and this question should be moved back to issues.
